caffeinated/admin/new/pricing/templates/event_tickets_datetime_edit_row.template.php

```diff
@@ -15,45 +15,46 @@
  * @var string  $trash_icon
  * @var string  $show_trash
  * @var string  $reg_list_url
+ * @var string  $dtt_reserved
  */
 ?>
-<tr valign="top" id="event-datetime-<?php echo $dtt_row; ?>"
+<tr valign="top" id="event-datetime-<?php echo absint($dtt_row); ?>"
     class="datetime-edit event-datetime-row edit-dtt-row ee-dtt-sortable">
     <td class="event-datetime-column date-name-column">
-        <input type="hidden" name="<?php echo $event_datetimes_name; ?>[<?php echo $dtt_row; ?>][DTT_ID]"
-               id="event-datetime-DTT_ID-<?php echo $dtt_row; ?>" class="event-datetime-DTT_ID"
-               value="<?php echo $DTT_ID; ?>">
-        <input type="hidden" name="<?php echo $event_datetimes_name; ?>[<?php echo $dtt_row; ?>][DTT_order]"
-               id="event-datetime-DTT_order-<?php echo $dtt_row; ?>" class="event-datetime-DTT_order"
-               value="<?php echo $DTT_order; ?>">
-        <input type="text" name="<?php echo $event_datetimes_name; ?>[<?php echo $dtt_row; ?>][DTT_name]"
-               id="event-datetime-DTT_name-<?php echo $dtt_row; ?>" class="ee-large-text-inp event-datetime-DTT_name"
-               value="<?php echo $DTT_name; ?>" placeholder="<?php esc_html_e('Add Title (optional)', 'event_espresso'); ?>">
+        <input type="hidden" name="<?php echo esc_attr($event_datetimes_name); ?>[<?php echo absint($dtt_row); ?>][DTT_ID]"
```

Comment:
should we maybe escape the entire name like you have done in other places?

ie:

```suggestion
        <input type="hidden" name="<?php echo esc_attr( "{$event_datetimes_name}[ {$dtt_row} ][DTT_ID]" ); ?>"
```

Reply:
Personally, I prefer the current method since imho I think it is more readable. But if you prefer the method you suggested then I will change them. There should be many instances to change.

Reply:
let's just leave it as is then